facebook/src/facebook_android.cpp

```
@@ -553,6 +573,21 @@ int Facebook_ShowDialog(lua_State* L)
     return 0;
 }
 
+void Platform_FetchDeferredAppLinkData(lua_State* L, int callback, int context, lua_State* thread)
+{
+    VerifyCallback(L);
+    g_Facebook.m_Callback_DeferredAppLink = callback;
+    g_Facebook.m_Self_DeferredAppLink = context;
+
+    JNIEnv* environment = Attach();
+    environment->CallVoidMethod(g_Facebook.m_FB, g_Facebook.m_FetchDeferredAppLink, (jlong)thread);
+
+    if (!Detach(environment))
+    {
+        dmLogError("An unexpected error occurred during Facebook JNI interaction.");
```

Why not return a lua error? (I see other functions that also have this issue)

E.g. "return luaL_error(L, "err msg");"

---

Fixed!